src/hooks/useSubmitQuery.ts

```diff
@@ -158,7 +200,7 @@ export const useSubmitQuery = () => {
         }
       }
 
-      const response = await post('chat_v2/', formData)
+      const response = await post('chat/', formData)
```

Comment:
https://ferox-games.slack.com/archives/D077EN52LCQ/p1724261115670829

Reply:
Thank you 🙏🏽